atc/db/migration/migrations/1570709099_add_varsources_to_pipelines.up.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+  ALTER TABLE pipelines ADD COLUMN var_sources text,
+                        ADD COLUMN nonce text;
```

Comment:
I actually thought over that when I added the `nonce` column. It should be named `foo_nonce`, but given there is only one column to encrypt, I though to keep consist to other tables, so that at least which looks better for now. As you have pointed the concern, let me rename the column, so that reduce one worm when you open the can.